meilisearch/src/routes/indexes/mod.rs

```diff
@@ -100,6 +100,7 @@ pub async fn list_indexes(
             Ok(Some(IndexView::new(uid.to_string(), index)?))
         })?;
     // Won't cause to open all indexes because IndexView doesn't keep the `Index` opened.
+    // error when trying to fix it: the trait `ExactSizeIterator` is not implemented for `Flatten<IntoIter<Option<IndexView>>>`
```

Comment:
I think we can remove this line if clippy is not flagging it, otherwise it might be confusing

```suggestion
```